charts/bar-api/values.yaml

```diff
@@ -8,24 +8,32 @@ postgresql:
     enabled: false
 
 java:
-  # Don't modify below here
   image: 'hmctspublic.azurecr.io/bar/api:latest'
   ingressHost: ${SERVICE_FQDN}
```

Comment:
Make it something like this :
https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.yaml#L3

Reply:
its done